### ISSUE TITLE
folder names when importing from bibtex

### ISSUE THREAD

**Issue by gpasqualetti** [papis/papis#325]
* papis version 0.11.1

I run
```
papis bibtex read mybib.bib import --all -o /mypath
```
Files get successfully imported, but folder names are of some `hash-author` form. In my papis config file I declared
```
add-folder-name = {doc[author_list][0][family]:.20}-{doc[year]}-{doc[title]:.6}
```
Is it possible to consistently apply this option when importing a bibliography?

**Comment by alejandrogallo** [comment]
You are right that this does not work right now. I think it should be implemented.

**Comment by gpasqualetti** [comment]
If you give me some hint about how to structure it I can try to implement it. I had a quick look at the code but I wasn't able to understand how to do it efficiently, since `bibtex import` essentially passes to the `add` command the hashed folder names as main argument. Any idea about the best way to implement a workaround?
